core/templates/dev/head/pages/library/search_bar_directive.html

```diff
@@ -25,7 +25,7 @@
       <span class="caret"></span>
     </button>
     <!-- The $event.stopPropagation() prevents the dropdown from closing after an option is selected. -->
-    <ul uib-dropdown-menu class="protractor-test-search-bar-dropdown-menu" role="menu" style="max-height: 400px; overflow: auto;" ng-click="$event.stopPropagation()">
+    <ul uib-dropdown-menu id="oppia-dropdown-bar" class="protractor-test-search-bar-dropdown-menu" role="menu" style="max-height: 400px; overflow: auto;" ng-click="$event.stopPropagation()">
```

Comment:
Add it as a class instead of id and rename to `oppia-search-bar-dropdown-menu`

Reply:
Hi @ankita240796 , Thanks for taking a look at PR.  I have made the changes as told, PTAL.